dwitter/templates/dweet/dweet.html

```diff
@@ -181,7 +182,14 @@
 
     var stats = createStats();
     setStatsVisibility({{ newDweet }});
-  
+
+    // Remember the default globals so that later we can remove any that
+    // were created by the dweet
+    var browserGlobals = {};
+    Object.keys(window).forEach(key => {
+      browserGlobals[key] = true;
+    });
```

Comment:
Some "functional programming" people recommend this style instead:
```js
var browserGlobals = Object.keys(window).reduce((a, key) => (a[key] = true, a), {});
```
or even:
```js
var browserGlobals = Object.keys(window).reduce((a, key) => ({...a, [key]: true}), {});
```
I could change to one of those, if you prefer. 😛

Reply:
Fun way of doing it; but I'd say leave it as is now. Easier to read for a greater number of people.